code/game/jobs/job/whitelisted.dm

```diff
@@ -36,6 +36,8 @@
 		trader_account = create_trader_account
 	M.mind.store_memory("<b>The joint trader account is:</b> #[trader_account.account_number]<br><b>Your shared account pin is:</b> [trader_account.remote_access_pin]<br>")
 
+	log_admin("([M.ckey]/[M]) started the game as a [job_title].")
```

Comment:
This one uses a different format, the others are `([ckey]) started...`. Unintended?

Reply:
Technically an oversight, but in the opposite way: the silicon logs I added don't follow the existing format, whereas this line does.

The trader started log is a simple copy paste from the current log_admin for regular jobs, complete with /[M]. See line 120 of /code/game/jobs/job/job.dm. This only seems to print the default name of a mob (ex: clowns, mimes, etc), and prints nothing for most jobs. This is an example log output of the current code:

`[22:49:01]ADMIN: (anglosphere/) started the game as a Atmospheric Technician.`
`[22:49:01]ADMIN: (loch2/) started the game as a Virologist.`
`[22:49:01]ADMIN: (cboyjet4/Silly Willy) started the game as a Clown.`
`[22:49:01]ADMIN: (dilt/) started the game as a Quartermaster.`
`[22:49:01]ADMIN: (blonichu/) started the game as a Roboticist.`

Which way should the logging be changed for consistency? I could simply add a / after the silicon's ckeys in this PR and it would appear as consistent in the logs.

Reply:
It doesn't matter, as long as it's consistent. I would go for `(ckey/name)` if obtaining the correct `name` doesn't require significant refactoring, otherwise just `(ckey)` is fine.

Reply:
I'd suggest we leave it as `(ckey)` for now.  The silicons do not even have their random names assigned at this point in the code (much less the name they may change to). Having it trigger later just to get a name would involve awkward guessing at station times and when the round started, or would require some sort of joined-at-roundstart variable being set.

Reply:
k